includes/ui/class-notice.php

```diff
@@ -32,10 +32,10 @@ public static function success( $args ) {
 
 		return self::render(
 			array_merge(
+				$args,
```

Comment:
If these are flipped it means that passing an icon change in $args will not take precedence. Is this somethihg we want?

Reply:
Good catch, thanks. Did this to fix the `classes` merges, tweaking it a bit more in ab543dd so the icon overwrites keep working too.